package.json

```diff
@@ -57,10 +58,10 @@
     "merge-stream": "^1.0.1",
     "mz": "^2.6.0",
     "plylog": "^0.4.0",
-    "polymer-analyzer": "^2.6.0",
-    "polymer-build": "^2.5.0",
-    "polymer-linter": "^2.2.0",
-    "polymer-project-config": "^3.10.0",
+    "polymer-analyzer": "^3.0.0-pre.14",
+    "polymer-build": "^3.0.0-pre.1",
+    "polymer-linter": "^3.0.0-pre.3",
+    "polymer-project-config": "^3.9.0",
```

Comment:
probably a merge issue, should keep 3.10.0 here

Reply:
+1, done